nameko/events.py

```diff
@@ -183,10 +168,69 @@ def __init__(self, source_service, event_type, handler_type=SERVICE_POOL,
         super(EventHandler, self).__init__(
             queue=None, requeue_on_error=requeue_on_error)
 
+    @property
```

Comment:
think i prefer `get_foo()` to make it clear(er) to a "caller" that this is a function call. esp. since it might be doing non-trivial stuff